packages/ui/src/components/toast/toast.tsx

```diff
@@ -86,7 +83,7 @@ const ToastTitle = forwardRef<
 >(({ className, ...props }, ref) => (
   <ToastPrimitives.Title
     ref={ref}
-    className={cn('text-sm leading-tight tracking-tight [&+div]:text-xs', className)}
+    className={cn('text-sm leading-tight tracking-[-0.02em] text-foreground-1', className)}
```

Comment:
ideally we should not be doing custom styles using `[]`. should we update `tracking-tight` to match the design expectation instead?